### PR TITLE
Add geo-coordinates for countries and states

### PR DESCRIPTION
**Pull request by fauvel**
- Add coordinates to `CM_Model_Location_State` and `CM_Model_Location_Country`
- Update them in `CMService_MaxMind`
- Retrieve the coordinates at a specified `$minLevel` in `CM_Model_Location::getCoordinates`

### CODE REVIEW

File: library/CM/Model/Location.php [hunk 1]
```diff
@@ -78,25 +78,34 @@ public function getAbbreviation($level = null) {
...
      */
-    public function getCoordinates() {
-        /** @var CM_Model_Location_Zip $location */
-        if ($location = $this->_getLocation(CM_Model_Location::LEVEL_ZIP)) {
-            return $location->getCoordinates();
-        }
-        /** @var CM_Model_Location_City $location */
-        if ($location = $this->_getLocation(CM_Model_Location::LEVEL_CITY)) {
-            return $location->getCoordinates();
+    public function getCoordinates($levelMin = null) {
```
Comment (fvovan) [inline library/CM/Model/Location.php]: Please modify test for it

File: library/CM/Model/Location.php [hunk 2]
```diff
@@ -392,12 +401,14 @@ public static function createAggregation() {
...
-     * @param string $abbreviation
+     * @param string     $name
+     * @param string     $abbreviation
+     * @param float|null $latitude
+     * @param float|null $longitude
      * @return CM_Model_Location
      */
-    public static function createCountry($name, $abbreviation) {
-        $country = CM_Model_Location_Country::create($name, $abbreviation);
+    public static function createCountry($name, $abbreviation, $latitude = null, $longitude = null) {
```
Comment (fvovan) [inline library/CM/Model/Location.php]: Please modify tests for updated create*** methods

**Comment by fauvel** [comment]
@fvovan please review!

**Comment by fauvel** [comment]
Tagged 1.223.0

**Pull request merged**